Lifecycle.md

```diff
@@ -68,31 +68,33 @@ For more information (and examples) of how to write a charter for a TL*,
 and other important getting-started documents, please see our 
 [examples](Bootstrap-Policies/).
 
-## Submitting an Application
+## Membership Application and Approval Process
 
-The application process to join the Planetary Software Organization
+The Planetary Software Organization is quite new and currently has
+limited resources available to mentor new projects. As such, projects
+are chosen for admission as mentors become available.
+
+You can apply at any time and the TSC and available mentors will
+help improve your application while awaiting available resources.
+
+The process to join the Planetary Software Organization
 as a TLP or TLWG is as follows.
 
+### 1. Application Completed - Pull Request Submitted
+
 Copy the [Application Template](Applications/Template.md), fill
 it out, add the file to the [Applications](Applications/)
-directory, and submit that as a Pull Request.
+directory and submit this as a Pull Request.
```

Comment:
I'm not a fan of this/that here. What about 'and submit the filled in Template.md as a Pull Request to this repository'?

Reply:
Added suggested text